packages/charts/test/chart-series.test.js

```diff
@@ -1,6 +1,7 @@
 import { expect } from '@vaadin/chai-plugins';
-import { aTimeout, fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
+import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
+import './chart-no-animation-styles.js';
```

Comment:
Could be named `not-animated-styles` to align with other components.

Reply:
Thanks, done.